Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CryptoManager.java

```diff
@@ -50,7 +54,7 @@
      */
     @Override
     public byte[] getBytesFromCertificates(X509Certificate[] arg0) throws WSSecurityException {
-        throw new WSSecurityException("getBytesFromCertificates");
+        throw new WSSecurityException(ErrorCode.FAILURE,"getBytesFromCertificates");
```

Comment:
Out of curiosity, what is the purpose of this class?  All I see are exceptions, null returns, or no-behavior functions.